plugins/apollo-explorer/src/components/ApolloExplorerBrowser/ApolloExplorerBrowser.tsx

```diff
@@ -56,9 +58,42 @@ export type ApolloEndpointProps = {
 
 type Props = {
   endpoints: ApolloEndpointProps[];
+  authCallback?: () => Promise<string>;
+  apiHolder?: ApiHolder;
 };
 
-export const ApolloExplorerBrowser = ({ endpoints }: Props) => {
+export const handleAuthRequest = ({
+  legacyIncludeCookies,
+  authCallback,
+}: {
+  legacyIncludeCookies?: boolean;
+  authCallback: any;
+}): HandleRequest => {
+  let cookies = {};
+  if (legacyIncludeCookies) {
```

Comment:
Hmm, we're adding this option but hardcoding it to `false` below. Could just drop this bit right?